bpf/net/bpf_filter.c

```diff
@@ -618,7 +618,7 @@ bpf_filter(pc, p, wirelen, buflen)
 int
 bpf_validate(f, len)
 	const struct bpf_insn *f;
-	int len;
+	u_int len;
```

Comment:
I fixed this differently in 406cfb76981a17b900cae43d3bbab7b5dae75900; it's probably best not to change the signature of functions that are part of the API, even if it *should* originally have been an unsigned length.